app/screens/channel_info/channel_info.js

```diff
@@ -363,41 +387,50 @@ export default class ChannelInfo extends PureComponent {
                 </View>);
         }
 
-        return (<View>
-            <ChannelInfoRow
-                action={this.handleFavorite}
-                defaultMessage='Favorite'
-                detail={this.state.isFavorite}
-                icon='star-o'
-                textId={t('mobile.routes.channelInfo.favorite')}
-                togglable={true}
-                theme={theme}
-            />
-            <View style={style.separator}/>
-            <ChannelInfoRow
-                action={this.handleMuteChannel}
-                defaultMessage='Mute channel'
-                detail={this.state.isMuted}
-                icon='bell-slash-o'
-                textId={t('channel_notifications.muteChannel.settings')}
-                togglable={true}
-                theme={theme}
-            />
-            {
-
-                /**
-                 <ChannelInfoRow
-                 action={() => true}
-                 defaultMessage='Notification Preferences'
-                 icon='bell-o'
-                 textId='channel_header.notificationPreferences'
-                 theme={theme}
-                 />
-                 <View style={style.separator}/>
-                 **/
-            }
-            {this.renderViewOrManageMembersRow() &&
-                <View>
+        return (
+            <React.Fragment>
+                <ChannelInfoRow
+                    action={this.handleFavorite}
+                    defaultMessage='Favorite'
+                    detail={this.state.isFavorite}
+                    icon='star-o'
+                    textId={t('mobile.routes.channelInfo.favorite')}
+                    togglable={true}
+                    theme={theme}
+                />
+                <View style={style.separator}/>
+                <ChannelInfoRow
+                    action={this.handleMuteChannel}
+                    defaultMessage='Mute channel'
+                    detail={this.state.isMuted}
+                    icon='bell-slash-o'
+                    textId={t('channel_notifications.muteChannel.settings')}
+                    togglable={true}
+                    theme={theme}
+                />
+                <View style={style.separator}/>
+                <ChannelInfoRow
+                    action={this.goToPinnedPosts}
+                    defaultMessage='Pinned Posts'
+                    image={pinIcon}
+                    textId={t('channel_header.pinnedPosts')}
+                    theme={theme}
+                />
+                {
+
+                    /**
+                     <ChannelInfoRow
+                     action={() => true}
+                     defaultMessage='Notification Preferences'
+                     icon='bell-o'
+                     textId='channel_header.notificationPreferences'
+                     theme={theme}
+                     />
+                     <View style={style.separator}/>
+                     **/
```

Comment:
Should we remove this commented block if not being used currently? (Looks like it was added 4 months ago related to archived channels)

Reply:
it is a functionality that we haven't added... I prefer to keep it there as a reminder.

@esethna can we look into adding Channel notification preferences for the next feature release?